keyboards/dumbpad/v3x/templates/keymap.c

```diff
@@ -0,0 +1,27 @@
+#include QMK_KEYBOARD_H
```

Comment:
This should be removed, and `encoder_update_kb` implemented in `v3x.c`instead

Reply:
Thanks for you fast feedback, done :)